Cargo.toml

```
@@ -11,7 +11,7 @@ categories = ["wasm", "gui", "web-programming"]
 
 [dependencies]
 log = "0.4"
-dioxus = { version = "0.4.0", default-features = false, features = ["macro", "html"] }
+dioxus = { version = "0.5.0-alpha.2", default-features = false, features = ["macro", "html"] }
```

let's bump it to stable!
```suggestion
dioxus = { version = "0.5", default-features = false, features = ["macro", "html"] }
```